CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.18.0
+### Added
+* Service: Use records to unpack the request parameters in `AgentRegistration`. [#2570](https://github.com/microsoft/onefuzz/pull/2570)
+* Service: Convert ADO traces to `customEvents` and update `notificationInfo`. [#2508](https://github.com/microsoft/onefuzz/pull/2508)
+* Agent: Include computer name in `AgentRegistration` & decode Instance ID from it. [#2557](https://github.com/microsoft/onefuzz/pull/2557)
+### Changed
+* Service: Improve webhook logging and accept more HTTP success codes. [#2568](https://github.com/microsoft/onefuzz/pull/2568)
+* CLI: Use the vitual env folder to store the config if it exists. [#2561](https://github.com/microsoft/onefuzz/pull/2561)
+### Fixed
+* Service: Reduce number of ARM calls in `ListVmss`. [#2539](https://github.com/microsoft/onefuzz/pull/2539)
+* Service: Exponential backoff for failed notifications. [#2555](https://github.com/microsoft/onefuzz/pull/2555)
+* Service: ETag updated in `Update` and `Replace`. [#2562](https://github.com/microsoft/onefuzz/pull/2562)
+* Service: Don't log an error if we delete a Repro and it is already missing. [#2563](https://github.com/microsoft/onefuzz/pull/2563)
+* CLI: Fix onefuzz config path. [#2567](https://github.com/microsoft/onefuzz/pull/2567)
```

Comment:
this one can be merged with the 
`CLI: Use the vitual env folder to store the config if it exists` item, it was a bug fix for that PR